tests/mcp/cloudrun/resources/gcp_cloudrun.yml

```diff
@@ -0,0 +1,30 @@
+project_id: &project_id cloud-run-test-1409-1
+location_id: "europe-west1"
+
+components:
+  specs:
```

Comment:
`specs` should allow for multiple services to be deployed, so specs should have a key which is the micro service name (it could just be one service) but it could equally be a few services. This would then warrant a `common` section